frontend/src/metabase/visualizations/components/settings/ChartSettingOrderedSimple.tsx

```diff
@@ -59,23 +58,22 @@ export const ChartSettingOrderedSimple = ({
   };
 
   const getItemTitle = (item: SortableItem) => {
-    return items[item.originalIndex]?.name || "Unknown";
+    return items.find(i => i.key === item.key)?.name || "Unknown";
   };
 
   const handleOnEdit = (item: SortableItem, ref: HTMLElement | undefined) => {
-    const single = series[item.originalIndex];
     onShowWidget(
       {
         props: {
-          seriesKey: keyForSingleSeries(single),
+          seriesKey: keyForSingleSeries(item.key),
```

Comment:
since `item.key` is already a series key shouldn't this line be just this?
```suggestion
          seriesKey: item.key,
```

I have verified that breakout column settings stop working.
![Screenshot 2022-11-22 at 3 08 07 PM](https://user-images.githubusercontent.com/1937582/203259517-dd46854d-c946-4717-8509-8abe94436274.png)